ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -160,6 +160,11 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-openstack-etcd-scaling
+- as: e2e-openstack-dualstack-techpreview
+  minimum_interval: 48h
```

Comment:
Every 2 days seems a bit too much for my taste, especially with our not-too-high capacity at the moment. Maybe every week would be enough for now? And if at some point it breaks because of a missed patch, we can review it. But I think with the release build cadence, once a week would be enough for now.

Reply:
how about every 3 days or 4 days?
My thinking was that the sooner we identify issues the better given this is a new feature. Once we see it's stable I can change it for a longer interval

Reply:
We agreed that for now 4 days was a good compromise.